src/test/java/github/weichware10/analyse/DiagrammTest.java

```diff
@@ -39,18 +39,18 @@ public void createRelFreqImgAreaShouldWork() {
      */
     @Test
     public void createViewTimeDistrSouldWork() {
-        Diagramm dia1 = new Diagramm(new Data(ToolType.ZOOMMAPS, 1), new ConfigDiagramm());
+        Diagramm dia1 = new Diagramm(new TrialData(ToolType.ZOOMMAPS, "1", "3"), new ConfigDiagramm());
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 103).